src/components/About.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Box,
+  Container,
+  Typography,
+} from '@material-ui/core';
+import Banner from '../assets/banner.jpg';
+
+const useStyles = makeStyles((theme) => ({
+  wrapper: {
+    backgroundColor: '#e0e2e5',
+    height: '100vh',
+  },
+  aboutBanner: {
+    backgroundImage: `url(${Banner})`,
+    backgroundSize: 'cover',
+    backgroundRepeat: 'no-repeat',
+    backgroundPosition: 'center',
+    padding: theme.spacing(0.25),
+  },
+  aboutHead: {
+    marginTop: theme.spacing(10),
+    marginBottom: theme.spacing(10),
+    color: theme.palette.common.white,
+    textAlign: 'center',
+  },
+  title: {
+    fontFamily: 'Merriweather, serif',
+  },
+  aboutContent: {
+    marginTop: 20,
+    fontFamily: 'Nanum Gothic, sans-serif',
+  },
+  changes: {
+    fontFamily: 'Merriweather, serif',
+    marginTop: 20,
+    marginBottom: 4,
+  },
+  link: {
+    color: '#1e3652',
+    padding: theme.spacing(0.125),
+    borderBottom: '1px solid black',
+    textDecoration: 'none',
+  },
+}));
+
+const About = () => {
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.wrapper}>
+      <Box className={classes.aboutBanner}>
+        <Container className={classes.aboutHead}>
+          <Typography className={classes.title} variant="h2">About</Typography>
+        </Container>
+      </Box>
+      <Container className={classes.aboutContent}>
+        <Typography variant="h6">
+          JSON-e is a data-structure parameterization system for embedding context in JSON objects.
+          <br />
```

Comment:
This page looks sort of "run together".  Would the `<p>` element work better?  Or is there some other approach?  It's fine to address this in a follow-up..

Reply:
Yeah! `<p>` tag would be better. In fact, it should be a `<p>` tag. I'll fix it. 

> This page looks sort of "run together".

I didn't get it.
Agree, we can work on styling and theme at the end.